frontend/.env.production

```diff
@@ -0,0 +1,3 @@
+REACT_APP_API_URL=http://localhost:1337/api/v1
```

Comment:
rename to `.end.dev` since `REACT_APP_AUTH_URL_ADMIN ` and `REACT_APP_AUTH_URL` should only exist in dev mode